tutorials/06-physics-simulation-concepts.md

```diff
@@ -0,0 +1,250 @@
+
+For rotation movement, we select the third icon from the left in the top left
+toolbar,  the Transform Control plugin, or by pressing the keyboard shortcut: R.
+The red circle represents roll, green is pitch, and blue is yaw. Click and hold
+on any of the circles while moving your mouse to rotate the entity around that axis.
+
+<img src="https://user-images.githubusercontent.com/18066876/97185874-e63e9780-17a0-11eb-955b-20200c3be4e4.gif"/>
+
+#### Input model pose on Component Inspector
+
+If exact pose input is required, we select the car model
+(e.g. `vehicle_blue`), then select the `Pose` drop down list. We can input the
+desired pose, for example `X: 0.0, Y: 1.0, Z: 2.0, Roll: 1.0, Pitch: 0.0, Yaw: 1.0`
+as follow:
+
+<img src="https://user-images.githubusercontent.com/18066876/97185954-fe161b80-17a0-11eb-8777-9e667889c468.gif"/>
+
+Regardless of the simulation is pausing or not, the car model will jump to the
+desired pose. For more detail about specifying pose, see this [tutorial](http://sdformat.org/tutorials?tut=specify_pose&cat=specification&).
+
+### Monitoring model pose
+
+To see the pose changing when the simulation running, we publish a
+\ref ignition::msgs::Twist "Twist" message to command the `vehicle_blue` car to
+move in a circle with 2 meters radius as follow:
+
+```bash
+ign topic -t "/model/vehicle_blue/cmd_vel" -m ignition.msgs.Twist -p "linear: {x: 1.0}, angular: {z: 0.5}"
+```
+
+This command tells the car to move in its coordinate frame with velocity
+1.0 meter per second in the X-axis and angular velocity 0.5 radians per second in
+Z-axis. We can monitor the model pose by selecting the moving car and then
+select the drop-down list `Pose`. Moreover, we could also read the model
+links' poses relative to their parent link by selecting the
+corresponding link on the model tree:
+
+<img src="https://user-images.githubusercontent.com/18066876/97189395-d45ef380-17a4-11eb-9c29-cf33b5b184c4.gif"/>
+
+Note that using the model tree as shown in the above gif, we can view the
+parameters and properties such as `visual` or `collision` of each link or joint
+in the model. For more detail about the kinematics of the model tree, see this [tutorial](http://sdformat.org/tutorials?tut=spec_model_kinematics&cat=specification&).
+
+### Visualizing collision
+
+One of the most wonderful features of physics simulation is the capability to
+simulate collision. We can do a fun experiment like this: while the `vehicle_blue`
+car is moving in a circle, we move the `vehicle_green` to be on `vehicle_blue`'s
+upcoming path. We will see the blue car will push the green car!
+
+<img src="https://user-images.githubusercontent.com/18066876/97190565-28b6a300-17a6-11eb-9ed5-0d6d50c15ad4.gif"/>
+
+To see where these collision parameters are set and how it works, please see
+this [tutorial](http://sdformat.org/tutorials?tut=spec_shapes&cat=specification&).
+
+## Physics concepts in SDFormat
```

Comment:
I think the next few sections about SDFormat isn't closely related to ign-physics and most of them can be referred back to sdformat documentation. I think it'd be nicer if we can focus more on the existing demo worlds.

Some example worlds that might be worth demonstrating here:
1. [lift_drag](https://github.com/ignitionrobotics/ign-gazebo/blob/ign-gazebo4/examples/worlds/lift_drag.sdf) shows how joint force, torque and pressure is supported in ign-physics, source code see https://github.com/ignitionrobotics/ign-gazebo/blob/ign-gazebo4/src/systems/lift_drag/LiftDrag.cc
2. [video_record_dbl_pendulum](https://github.com/ignitionrobotics/ign-gazebo/blob/ign-gazebo4/examples/worlds/video_record_dbl_pendulum.sdf) shows how inertia and gravity affect the object movement by showing free swing of the pendulum 
3. [multicopter_velocity_control](https://github.com/ignitionrobotics/ign-gazebo/blob/ign-gazebo4/examples/worlds/multicopter_velocity_control.sdf) shows how ign-physics supports gravity, actuators and inertia to control object velocity, source code https://github.com/ignitionrobotics/ign-gazebo/blob/ign-gazebo4/src/systems/multicopter_control/MulticopterVelocityControl.cc
4. [trisphere_cycle_wheel_slip](https://github.com/ignitionrobotics/ign-gazebo/blob/ign-gazebo4/examples/worlds/trisphere_cycle_wheel_slip.sdf.) This is a fun one. This one shows the slip compliance and collision supported by ign-physics with source code https://github.com/ignitionrobotics/ign-gazebo/blob/ign-gazebo4/src/systems/wheel_slip/WheelSlip.cc

Reply:
Sorry I messed up a bit :), I exclude the Wheel Slip for now because it reports some errors. I also include Bouyancy because it seems important.